app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -263,7 +275,7 @@ class NewMessageViewModel @Inject constructor(
 
         populateWithExternalMailDataIfNeeded(draft = this, intent)
 
-        body = getWholeBody()
+        body = getWholeBody("", uiSignature, uiQuote)
```

Comment:
Here, the `uiBody` isn't necessary empty _(ex: mailTo with payload)_.

Why hardcoding an empty string instead of sending the `uiBody` ?

Reply:
I think the reason why I placed an empty string is because `body` is never used in this situation, the whole line is useless. Even when there's a mailto with payload we never end up using the value of the field `body` which is set here. And setting it for no reason here, wrongly makes us believe that setting `body` using values like `uiBody` is required for the code to work properly and that we cannot stop relying on `uiBody` here.

But this is not the right way to show it anyway, you're right. We should not use the Draft's properties to pass a value like this around the `getExistingDraft()` function

Reply:
I've modified the code in this regard, `body`'s usages are now limited to what's really needed

Reply:
We are saving the Draft into Realm just after.
By removing this line, it means the `body` won't be saved in Realm.